modules/apps/app-builder/app-builder-web/src/main/resources/META-INF/resources/js/pages/entry/EditEntry.es.js

```diff
@@ -87,9 +94,20 @@ export const EditEntry = ({
 							.catch(onError);
 					}
 					else {
-						addItem(
-							`/o/data-engine/v2.0/data-definitions/${dataDefinitionId}/data-records`,
-							dataRecord
+						fetch(
+							createResourceURL(baseResourceURL, {
+								p_p_resource_id: '/app_builder/add_data_record',
+							}),
+							{
+								body: new URLSearchParams(
+									Liferay.Util.ns(namespace, {
+										appBuilderAppId: appId,
+										dataRecord: JSON.stringify(dataRecord),
+										dataRecordId,
```

Comment:
Hey @albinojunior, we do not need to pass the dataRecordId parameter here. ;)

Reply:
Ok @rafaprax. I have copied the implementation of the app-builder-workflow EditEntry. Should I remove it  from there too?

Thx!